### PR TITLE
Feature/data layer

### PR DESCRIPTION
**Pull request by 10dimensions**
- Query GeoData (Ceramic ID and Parcel ID) based on GPS coordinates

- Using ceramic id and parcel id, query for parcel info and parcel content

- Setup page layout

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,13 +1,13 @@
 import logo from './logo.svg';
 import './App.css';
 
-import GPS from './components/gps/GPS';
+import GoeWebSystem from './components/GeoWebSystem/GeoWebSystem';
```
Comment (codynhat) [inline src/App.js:4]: Looks like a small typo here.

Comment (codynhat) [reply src/App.js:4]: `GoeWebSystem` -> `GeoWebSystem`

File: package.json [hunk 2]
```diff
@@ -4,6 +4,7 @@
   "private": true,
   "dependencies": {
     "@apollo/client": "^3.3.13",
+    "@ceramicnetwork/cli": "^0.22.0",
```
Comment (codynhat) [inline package.json:7]: Does the cli package pull in the http-client? I believe the http-client is all that is needed.  `@ceramicnetwork/http-client`

**Review by codynhat — APPROVED** [review]
Looks great! One thing to take note of is we should think about which node the Ceramic HTTP client points to. Right now, it is our testnet node, which is fine. As Ceramic goes to mainnet, there has been discussion around gateway nodes optimized for reads vs nodes optimized for writes. This browser would only be reading, so I think we would want to point to the gateway nodes.

File: src/App.js (hunk 1 above)
Comment (10dimensions) [reply src/App.js:4]: Resolving it the next request. Alternatively, thinking of naming the component GWS (on the lines of GPS)

File: package.json (hunk 2 above)
Comment (10dimensions) [reply package.json:7]: Using only that particular package (http), to avoid confusion

**Pull request merged**